### PR TITLE
526 intl date

### LINKED ISSUE #526: Item date is displayed in english format MM/DD/YY even when language si set to french
Capture d’écran 2023-01-11 à 10 25 31

Also the date is cropped.

### PR DESCRIPTION
**Pull request by pyphilia**
close #526

### CODE REVIEW

File: src/components/main/DeleteItemDialog.tsx [hunk 1]
```diff
@@ -20,17 +20,33 @@ const descriptionId = 'alert-dialog-description';
...
+  );
 
   const onDelete = () => {
     if (itemIds.length > 1) {
       deleteItems(itemIds);
     } else {
-      deleteItem(itemIds);
+      // we still use this call because it has feedback
+      // remove when deleteItems will have feedback
+      deleteItem(itemIds as unknown as string);
```
Comment (spaenleh) [inline src/components/main/DeleteItemDialog.tsx:49]: Wouldn't it be better to send `itemIds[0]` instead ?

File: src/utils/date.ts [hunk 2]
```diff
@@ -1,9 +1,44 @@
+/* eslint-disable import/no-duplicates */
```
Comment (spaenleh) [inline src/utils/date.ts:1]: Do we need this ?

**Review by spaenleh — CHANGES_REQUESTED** [review]
🐷  Looks good to me, just a few comments, here and there and we should remove branch deps once the prs are merged 😃

File: src/components/main/DeleteItemDialog.tsx (hunk 1 above)
Comment (pyphilia) [reply src/components/main/DeleteItemDialog.tsx:49]: Actually it's either an array or a string.. not the best I know D:

**Review by spaenleh — APPROVED** [review]
🧈

**Pull request merged**